docs/advanced/UsingRunSaga.md

```diff
@@ -7,20 +7,29 @@ When a Saga is started (either at startup or later dynamically), the middleware
 `redux-saga` provides a way to run a Saga outside of the Redux middleware environment and connect it to a custom Input/Output.
 
 ```javascript
-import { runSaga } from 'redux-saga'
+import { runSaga, stdChannel } from 'redux-saga'
 
-function* saga() { ... }
+const emitter = new EventEmitter();
```

Comment:
let's remove the semicolons here 😃 . We've set `semi: false` in our prettier config.

Reply:
we should tweak out lintstaged config to format .md files too